src/main/java/net/sf/jabref/logic/util/date/EasyDateFormat.java

```diff
@@ -31,12 +54,17 @@ public String getCurrentDate() {
      *
      * @return The formatted date string.
      */
-    public String getDateAt(Date date) {
+    public String getDateAt(ZonedDateTime dateTime, boolean isoFormat) {
         // first use, create an instance
-        if (dateFormatter == null) {
-            String format = Globals.prefs.get(JabRefPreferences.TIME_STAMP_FORMAT);
-            dateFormatter = new SimpleDateFormat(format);
+        if ((dateFormatter == null) || (isoFormat != isISOFormat)) {
+            if (isoFormat) {
+                dateFormatter = DateTimeFormatter.ISO_LOCAL_DATE;
+            } else {
+                String format = Globals.prefs.get(JabRefPreferences.TIME_STAMP_FORMAT);
```

Comment:
Since we want to get away from using Globals.prefs in logic classes, I would propose to add constructors which takes the format as input. E.g.
````java
EasyDateFormat(String dateFormat)
   this(DateTimeFormatter.ofPattern(dateFormat))
EasyDateFormat(DateTimeFormatter dateFormat)
   this.dateFormat = dateFormat;
static EasyDateFormat fromPreferences(JabRefPreferences preferences)
    return new EasyDateFormat(preferences.get(JabRefPreferences.TIME_STAMP_FORMAT));
static EasyDateFormat IsoDateFormat = new EasyDateFormat(DateTimeFormatter.ISO_LOCAL_DATE)
````